governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024
+summary: Signal your support or opposition to the token and product launch parameters.
+discussion_link: https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-09-09T16:00:00
+end_date: 2024-09-12T16:00:00
+---
+# Poll: Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Ecosystem team. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, September 9 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+**This proposal will only take effect if the concurrent Atlas v2 Upgrade proposal passes.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes. In summary, this vovers the following items (find attached corresponding GitHub Repos, which include README files that explain all technical details):
+
+* New Token contracts and corresponding token converters
```

Comment:
```suggestion
* New Token contracts and corresponding token converters initializtion.
```

Reply:
I think this list is generally a bit inconsistent in form. There are:

- verbal forms ("Use USDS to get SKY").
- nominalized forms ("SBE Upgrade")
- Some of the nominalized forms are 100% noun compounds, some are not ("The introduction of STR" in contrast to "STR introduction").

Additionally, list items are missing a period at the end and capitalization is inconsistent.

Reply:
In general we have tried to keep the wording as close to the post from BA Labs as possible. This is to ensure there is no ambiguity if the text differs between the two sources.